### PR TITLE
594 Minor content changes

### PR DESCRIPTION
**Pull request by jema28**
#### Issues this PR relates to

#594

### CODE REVIEW

**Comment by jema28** [comment]
@thejoefriel I've had to reduce the body text to 14px to make it fit as it was still too long.

**Comment by jema28** [comment]
Will sort the rest of the changes now.

**Comment by jema28** [comment]
@thejoefriel I've made the other content changes, now ready for review.

File: client/src/pages/GiveCredits/earnActivityCategories.js [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
 const earnCategories = [
   {
-    category: "adminOfficeSupport",
-    categoryName: "Admin & Office Support",
+    category: "officeAdminSupport",
+    categoryName: "Office & Admin Support ",
     subtitle: "e.g. working on reception or completing paperwork",
   },
   {
-    category: "developingServices",
```
Comment (thejoefriel) [inline client/src/pages/GiveCredits/earnActivityCategories.js:10]: Why have all the categories been changed as well? This means that the icons are no longer working for the ones you've changed: http://www.clipular.com/posts/4892976940318720?k=WytxaXpXqwqZhDyWTSXcsaMVn_s

Please change that back unless there's a reason?

**Comment by thejoefriel** [comment]
> @thejoefriel I had to reduce the body text to 14px to make it fit as it was still too long.

OK cool. @harrietepugh due to there being more text in the welcome screens we've had to bring the font size down slightly. Are you ok with this? https://www.loom.com/share/65100693477b4f8b8cf9de21f9c5d4a0

**Comment by harrietepugh** [comment]
@thejoefriel @jema28 can we make the images smaller? Reluctant to make font smaller as this came up in testing! Lots of older people found it small as it is!

**Comment by jema28** [comment]
Hi @thejoefriel, I've made those changes, ready for review:
- reverted the categories back to the originals
- changed earn group nav link and reverted headline to original (changed it by mistake)
- reduced the size of the image to 230px instead of 250px

**Pull request merged**